go-controller/pkg/config/config.go

```diff
@@ -152,6 +152,7 @@ var (
 			V4OVNServiceHairpinMasqueradeIP: net.ParseIP("169.254.169.5"),
 			V6OVNServiceHairpinMasqueradeIP: net.ParseIP("fd69::5"),
 		},
+		UserDefinedNetworkConntrackMarkBase: 2,
```

Comment:
why is the default 2? 0 network-id is default network right? who ate 1?

Reply:
Right, moving to 0, I doubt we may need to modify that.

Reply:
moving to 1 or 0?

Reply:
Since this is the base, it should be `0`, since the next value will be `1`.